user_sync/app.py

```diff
@@ -121,6 +121,7 @@ def init_log(logging_config):
     builder.set_bool_value('log_to_file', False)
     builder.set_string_value('file_log_directory', 'logs')
     builder.set_string_value('file_log_level', 'info')
+    builder.set_string_value('file_log_name', 'default')
```

Comment:
same change as above: parameter is `file_log_name_format` and default value is `'{%Y-%m-%d}.log'`